migrations/20231120125930-remove-pledges.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    console.log('Remove pledges (orders)');
+    await queryInterface.sequelize.query(`
+      UPDATE "Orders"
+      SET
+        "deletedAt" = COALESCE("deletedAt", NOW()),
+        "status" = 'EXPIRED',
```

Comment:
I hesitated between `EXPIRED` and `CANCELLED`, but the second seems exclusively linked to contributions